views/users/new.hbs

```diff
@@ -1,3 +1,4 @@
+<p>This is the sign up page</p>
```

Comment:
Good to have a clear title

Reply:
Nice, makes it much easier to differentiate between code